.dockerignore

```diff
@@ -0,0 +1,13 @@
+build/
+configure.ac
+COPYING*
+docs/
+examples/
+INSTALL*
+Jenkinsfile
+Makefile.am
+m4/
+README.md
+test/
+Testing/
```

Comment:
I don't know what "Testing" is, is this from your environment?

Reply:
nah, i just looked at the directory listing, but if this is not existing (anymore) or some other reason, it should be indeed gone